<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room</class>
 <widget class="QWidget" name="Room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="MainRoom">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>190</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Welcome To Our Project (Huy + Dang + Khoi)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createRoom">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Room</string>
     </property>
    </widget>
    <widget class="QPushButton" name="joinRoom">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>340</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Join Room</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ListRoom"/>
   <widget class="QWidget" name="RoomDetail">
    <widget class="QLabel" name="questionImage">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>471</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/waiting_image.webp</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="leaveRoom">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Leave</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="chatView">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>571</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="timer">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 24px; color: black;</string>
     </property>
     <property name="text">
      <string>Time: 9s</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputAnswer">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>550</y>
       <width>451</width>
       <height>28</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Input your answer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submitAnswer">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>550</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLabel" name="playerOneAvt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>C:/Users/USER/Desktop/30soccer-ronaldo-1-76fd-mobileMasterAt3x.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="playerTwoAvt">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>490</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>C:/Users/USER/Desktop/avatar-1606916__341.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="userOne">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>User 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="userTwo">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>460</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="pointOne">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Point: 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="pointTwo">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>430</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Point: 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="questionIndex">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 16px; color: black;</string>
     </property>
     <property name="text">
      <string>Question: 0</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="roomId">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Room id:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
